v2/captions_test.go

```diff
@@ -1,32 +1,32 @@
-package threeplay_test
+package v2_test
 
 import (
-	"testing"
-
-	"github.com/nytimes/threeplay"
+	"github.com/nytimes/threeplay/common"
+	"github.com/nytimes/threeplay/v2"
 	"github.com/stretchr/testify/assert"
-	gock "gopkg.in/h2non/gock.v1"
+	"gopkg.in/h2non/gock.v1"
+	"testing"
```

Comment:
Run ``goimports`` to rearrange the imports in 2 separate groups.

Reply:
I have it on, but it doesn't always work 🤦‍♀